docs/manage-jobs-in-gravitino.md

```diff
@@ -0,0 +1,583 @@
+---
+title: "Manage jobs in Gravitino"
+slug: /manage-jobs-in-gravitino
+date: 2025-08-13
+keywords: job, job template, gravitino
+license: "This software is licensed under the Apache License version 2."
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Introduction
+
+Starting from 1.0.0, Apache Gravitino introduces a new submodule called job system for users to 
+register, run and manage jobs. This job system collaborates with the existing metadata 
+management, brings users the ability to execute the jobs or actions based on the metadata, which 
+we called metadata-driven actions, for example, running a job to compact some Iceberg tables, 
+running a job to clean old data based on the TTL properties, etc.
+
+The aim of the job system is to provide a unified way to manage job templates and jobs,
+including registering job templates, running jobs based on the job templates, etc. The job
+system itself is a unified job submitter, allowing users to run jobs through the job system
+itself, but it doesn't provide the actual job execution capabilities. Instead, it relies on the
+existing job executors (schedulers) like Apache Airflow, Apache Livy to execute the jobs. 
+Gravitino's job system provide an extensible way to connect to different job executors.
+
+:::note 
+1. The job system is a new feature introduced in Gravitino 1.0.0, and it is still under
+   development so some features may not be fully implemented yet.
+2. The aim of the job system is not to replace the existing job executors. So, it can only 
+   support running single job at a time, and it doesn't support job scheduling for now.
```

Comment:
I can't understand it clearly. The Gravitino job system provides APIs for submitting jobs to external job executors or schedulers, such as Dolphin Scheduler or Airflow.  

I can't get the sentence `it can only support running single job at a time`, do you mean that currently we only support `LocalJobExecutor`?

Reply:
I meaning we only support running a single job, but most of the schedule supporting running a DAG with multiple jobs.